sqlalchemy_hana/dialect.py

```diff
@@ -240,7 +292,16 @@ def get_isolation_level(self, connection):
         return result[0]
 
     def _get_server_version_info(self, connection):
-        pass
+        """Get the version of the HANA server used by a connection.
+
+        Returns a tuple of (`major`, `minor`, `service pack`, `patch`, `build`)
+        """
+        db_version = connection.execute("SELECT VERSION FROM SYS.M_DATABASE").scalar()
```

Comment:
Does this really work? If yes, I think it would be good to write a test for it.

Reply:
Yep, seems to work well. In fact, for a while some tests would not run without it until the fix here: https://bitbucket.org/zzzeek/sqlalchemy/issues/4249/error-when-testing-dialect-with-an

I'll put together a test for it